calculator.js

```diff
@@ -1,28 +1,69 @@
+// const printInputs = function(num1, num2)
 
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+const addCalc = function(num1, num2) {
+  result = (num1 * 1) + (num2 * 1)
+  console.log(`${num1} + ${num2} = ${result}`)
+};
+
+const subCalc = function(num1, num2) {
+  result =  (num1 * 1) - (num2 * 1)
+  console.log(`${num1} - ${num2} = ${result}`)
+};
+
+const multCalc = function(num1, num2) {
+  result = (num1 * 1) * (num2 * 1)
+  console.log(`${num1} * ${num2} = ${result}`)
+};
+
+const divCalc = function(num1, num2) {
+  if (!(num2 * 1)){
+    console.log("You cannot divide by zero")
+  } else 
+    result = (num1 * 1) / (num2 * 1)
+    console.log(`${num1} / ${num2} = ${result}`)
 }
-// '3 + 5 = 8'
 
-const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
-
-  // Questions to ask and answer:
-  // What is promptInput?
-  // What data type? What does it hold? What does it represent?
-  // How do we read values from it? What syntax?
-  // How can we use it?
-  // Can we call our existing functions now, inside of this function?
+const modCalc = function(num1, num2) {
+  const result =  (num1 * 1) % (num2 * 1)
+  console.log(`${num1} * ${num2} = ${result}`)
+};
+
+const validNum = function(num1, num2 ) {
+  if(isNaN(num1) || isNaN(num2)) {
+    if(isNaN(num1)) {
+      console.log(`${num1} is not a number.`)
+    }
+    if(isNaN(num2)) {
+      console.log(`${num2} is not a number.`)
+  } 
+  } else {
+    return true
+  }
 }
 
-// Example manual testing of calculator.  
-calculateUserInput({}, {
-  num1: 3,
-  num2: 4,
-  operation: 'add',
-});
+const calculateUserInput = function (error, promptInput) {
+
+  const operation = promptInput.operation
+  const num1 = promptInput.num1
+  const num2 = promptInput.num2
+  if (!validNum(num1, num2)){
+    return 
+  }
+ 
+    if(operation === 'add' || operation === '+'){
+      return addCalc(num1, num2)
+    } else if (operation === 'subtract' || operation === '-'){
+      return subCalc(num1, num2)
+    } else if (operation === 'multiply' || operation === '*'){
+      return multCalc(num1, num2)
+    } else if (operation === 'divide' || operation === '/'){
+      return divCalc(num1, num2)
+    } else if (operation === 'modulo' || operation === '%'){
+      return modCalc(num1, num2)
+    } else {
+      console.log('unsupported operator')
+    }
```

Comment:
scooch these conditionals over to the left a bit more. They should be level with lines 46-51